sdk/storage/perf-tests/storage-blob/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../../../tsconfig.package",
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
`target: es5` but `lib: es5, es6, es7, esnext` feels like a huge mismatch.

Reply:
I just copied the tsconfig that we have for `storage-blob` SDK.

Reply:
Looks like I need `"lib": ["dom", "esnext"]`, removed the rest.